web-client/src/app/views/wallet/wallet.page.ts

```diff
@@ -95,6 +97,7 @@ export class WalletPage implements OnInit {
   }
 
   async ionViewWillEnter() {
+    this.bookmarkService.getBookmarks();
```

Comment:
Given this is an `async` function, shouldn't the `await` keyword be  used before calling the function.

`await this.bookmarkService.getBookmarks();`

Reply:
You use `await` if you need to use the results right away, but we don't for this case. We call  it like that so that the result are available in the store when we open the transaction history. 

If we needed the results on the wallet page, yes, we should probably use await, but we don't.

Reply:
I see, thanks for the clarification. The PR looks good to go, the fix works!